components/notifier/src/destinations/ms_teams.py

```diff
@@ -40,9 +39,10 @@ def create_connector_card(destination: str, notification: Notification) -> pymst
 
 def send_notification(destination: str, notification: Notification) -> None:
     """Send notification to Microsoft Teams using a Webhook."""
-    logging.info("Sending notification to configured teams webhook")
+    logger = get_logger()
+    logger.info("Sending notification to configured teams webhook")
     card = create_connector_card(destination, notification)
     try:
         card.send()
     except Exception:
-        logging.exception("Could not deliver notification")
+        logger.exception("Could not deliver notification")
```

Comment:
Wow, that's exotic. Is there an added benefit to using `exception` over `error`?

Reply:
That you haven't encountered it before doesn't make it exotic :-)

See https://docs.python.org/3/library/logging.html#logging.Logger.exception